maps/atlas.dmm

```diff
@@ -2529,8 +2531,7 @@
 	light_type = /obj/item/light/tube/reddish;
 	pixel_y = 21
 	},
-/obj/storage/secure/crate/plasma/armory/anti_biological,
```

Comment:
I thought the contraband locker being in the main security room was nicer than it being in the little supply room, so I moved the anti-biological crate there to make space for the locker.